client/main.js

```diff
@@ -0,0 +1,51 @@
+import { render } from "react-dom";
+
+import Empirica from "meteor/empirica:core";
+
+import Round from "./game/Round";
+import Consent from "./intro/Consent";
+import InstructionStepOne from "./intro/InstructionStepOne";
+import InstructionStepTwo from "./intro/InstructionStepTwo";
+import Quiz from "./intro/Quiz";
+import ExitSurvey from "./exit/ExitSurvey";
+import Thanks from "./exit/Thanks";
+
+// Set the Consent Component you want to present players (optional).
+Empirica.consent();
```

Comment:
This should generally be commented out if you're not changing it. In this case, I think it maybe should be:

```suggestion
Empirica.consent(Consent);
```

...since `Consent` from "./intro/Consent" is loaded into this file.